packages/cxl-lumo-styles/scss/global.scss

```diff
@@ -8,6 +8,7 @@ html {
   --cxl-wrap-padding: var(--lumo-space-m);
   --cxl-space-sm: 12px;
   --cxl-color-light-gray: hsla(0, 0%, 96%, 1);
+  --cxl-color-athens-gray: hsla(210, 20%, 96%, 1);
```

Comment:
Maybe not to action now, but I think these are being added to the wrong place. There's a colors.scss file, in which we should use theme lumo variables when possible, but should also have the extra variables.